src/engine/enginebuffer.cpp

```diff
@@ -1235,7 +1235,7 @@ void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
 
     // Update indicators that are only updated after every
     // sampleRate/kiUpdateRate samples processed.  (e.g. playposSlider)
-    if (m_iSamplesCalculated > (m_pSampleRate->get() / kiPlaypositionUpdateRate)) {
+    if (m_iSamplesCalculated > (kSamplesPerFrame * m_pSampleRate->get() / kiPlaypositionUpdateRate)) {
```

Comment:
I fixed this commit to only fix the documentation, and added a separate commit to change the update rate from 20 Hz to 15 Hz (instead of 10 Hz). Does that work?